modules/apps/analytics/analytics-settings-web/src/main/resources/META-INF/resources/js/pages/wizard/PropertyStep.spec.tsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import '@testing-library/jest-dom/extend-expect';
+import {act, render} from '@testing-library/react';
+import React from 'react';
+
+import PropertyStep from './PropertyStep';
+
+const response = {
+	actions: {},
+	facets: [],
+	items: [
+		{
```

Comment:
Just for curiosity: do you need all this objects inside `items` on your test? I'm not seeing where you're using them on your test